handlers/handlers.go

```diff
@@ -3,16 +3,17 @@ package handlers
 import (
 	"sync/atomic"
 
-	"github.com/gorilla/mux"
+	"github.com/go-chi/chi/v5"
 )
 
 // Router register necessary routes and returns an instance of a router.
-func Router() *mux.Router {
-	r := mux.NewRouter()
+func Router() *chi.Mux {
+
+	r := chi.NewRouter()
 	isReady := &atomic.Value{}
 	isReady.Store(true)
 
-	r.HandleFunc("/", home).Methods("POST")
+	r.Post("/", home)
 	r.HandleFunc("/healthz", healthz)
```

Comment:
May be replace handlefunc with get? 
The health and ready endpoints should only respond to get method.

Reply:
It works but let's see with get methods